service/initialise.go

```diff
@@ -27,3 +30,42 @@ var GetHealthCheck = func(cfg *config.Config, buildT, commit, ver string) (Healt
 	hc := healthcheck.New(versionInfo, cfg.HealthCheckCriticalTimeout, cfg.HealthCheckInterval)
 	return &hc, nil
 }
+
+// GetKafkaConsumer creates a Kafka consumer
+var GetKafkaConsumer = func(ctx context.Context, cfg *config.Config) (kafka.IConsumerGroup, error) {
+	cgChannels := kafka.CreateConsumerGroupChannels(1)
```

Comment:
I would suggest replacing the `1` with cfg.NumWorkers (so that the channel buffer has the same size as the number of go-routines consuming from it)